app/views/course/leaderboards/show.html.slim

```diff
@@ -1,4 +1,4 @@
-= page_header
+= page_header(@settings.title)
```

Comment:
The helper function uses the default translation if title is nil 

https://github.com/Coursemology/coursemology2/blob/master/app/helpers/application_helper.rb#L35